meson.build

```diff
@@ -466,11 +466,9 @@ _tools = [
   'rmmod',
 ]
 
-kmod_symlink = find_program('scripts/kmod-symlink.sh')
 foreach tool : _tools
   if get_option('tools')
-    symlink = join_paths(bindir, tool)
-    meson.add_install_script(kmod_symlink, symlink)
+    install_symlink(tool, pointing_to: 'kmod', install_dir: bindir)
```

Comment:
do we have the exact same tree output?

Reply:
Just had another look and they seems identical.